.github/workflows/build.yml

```diff
@@ -158,6 +158,10 @@ jobs:
                 name: Validate database schema
                 run: (cd tests/Application && bin/console doctrine:schema:validate)
 
+            -
+                name: Run ECS
+                run: vendor/bin/ecs check src/ spec/ tests/Behat tests/Integration
```

Comment:
In the ecs.php file we skip all *Spec.php files. We should remove that or remove "spec/" from here.